src/NumberTheory/QuadFormAndIsom/enumeration.jl

```diff
@@ -202,18 +261,28 @@ function is_admissible_triple(
     return false
   end
 
-  qA = discriminant_group(A)
-  qB = discriminant_group(B)
 
   # At this point, if C is unimodular at p, the gluing condition is equivalent to having
   # an anti-isometry between the p-part of the (quadratic) discriminant forms of A and B
-  if iszero(valuation(det(C), p))
-    return is_anti_isometric_with_anti_isometry(primary_part(qA, p)[1], primary_part(qB, p)[1])[1]
-  end
-
   l = valuation(level(C), p)
   Ap = local_symbol(A, p)
   Bp = local_symbol(B, p)
+  la = valuation(level(Ap), p)
+  lb = valuation(level(Bp), p)
+
+  # qA = discriminant_group(A)
+  # qB = discriminant_group(B)
+  if iszero(valuation(det(C), p))
+    # this should be superflous, because there exists a unimodular overlattice already?
+    #Ap = local_symbol(A, p)
+    #Bp = local_symbol(B, p)
+    
+    # fl0 =  is_anti_isometric_with_anti_isometry(primary_part(qA, p)[1], primary_part(qB, p)[1])[1]
+    fl1 = _is_anti_isometric_quadratic(Ap, Bp, l+1)
+    #@assert fl0 == fl1
```

Comment:
```suggestion
    # The lattice C_p is unimodular, so the level of A_p and B_p is at most p
    fl1 = _is_anti_isometric_quadratic(Ap, Bp, 1)
```
I guess it is easier to follow like that, at least to me. Since $C_p$ is unimodular, `l` is 0 and the previous line is indeed equivalent to the discriminant groups of $A_p$ and $B_p$ being anti-isometric as quadratic modules